src/Simulator.cpp

```diff
@@ -46,11 +46,17 @@ void Simulator::simulateHover()
         // completely made up magnitude, but it will go in the right direction
         gps->updateAltitude(double((rpms - RPM_EQUILLIBRIUM) / 100));
     }
+
+    // if we've reached the target takeoff altitude, return to equilibrium
+    if (ctrl->getTakeoffAlt() == gps->getAltitude())
+    {
+      ctrl->updateMotors(RPM_EQUILLIBRIUM);
+    }
```

Comment:
This is the right idea, but in the wrong place.  The simulator is supposed to be outside the project itself (it;'s there to provide a test harness) and shouldn't be giving flight controls to the system.  I would do this check either in a new thread that is the flight controllers computer, or in the TrifanHarness as part of the interaction for takeoff.  The simulator's sole job is to look at how the controller is flying, and decide what should happen to the GPS.